packages/core/src/components/forms/text-inputs.md

```diff
@@ -0,0 +1,85 @@
+@# Text inputs
+
+Blueprint provides two ways to create a text input:
+
+1. CSS only: apply `Classes.INPUT` to an `<input>` or `<textarea>` element to
+   style the native HTML tag.
+1. React component: use the `InputGroup` component for an advanced single-line
```

Comment:
This should be "2."

Reply:
markdown auto-increments when rendering. i prefer making them all the same number so it's easier to edit later.